specification/flag-evaluation/flag-evaluation.json

```diff
@@ -0,0 +1,164 @@
+[
```

Comment:
Accidentally checked in JSON? .gitignore?

Reply:
I wasn't sure if I should commit this or not.

Reply:
Up to you. Maybe create a CI step that runs the generator and fails if there are any files in `git diff` which would ensure that the JSON is always up to date.

Reply:
I've opened: https://github.com/open-feature/spec/issues/36